modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -13,6 +13,18 @@ definition {
 			locator1 = "PRMMDFRequest#ADD_BUTTON");
 	}
 
+	@summary = "Change MDF status"
+	macro changeMDFStatus {
+		var key_mdfName = "${mdfName}";
+		var key_button = "${statusButton}";
+
+		Click(locator1 = "PRMMDFRequest#ID");
+
```

Comment:
Hi @mateussandes23, I think that this path doesn't exist. Could you please double check?

Reply:
Also, I need that you create a separated macro to enter on MDF. 
This macro should be only to change the MDF status.